oci/layout/oci_dest_test.go

```diff
@@ -108,6 +109,27 @@ func TestPutManifestTwice(t *testing.T) {
 	assert.Equal(t, 2, len(index.Manifests), "Unexpected number of manifests")
 }
 
+func TestPutTwoDifferentTags(t *testing.T) {
+	ref, tmpDir := refToTempOCI(t)
+	defer os.RemoveAll(tmpDir)
+
+	ociRef, ok := ref.(ociReference)
+	require.True(t, ok)
+
+	putTestConfig(t, ociRef, tmpDir)
+	putTestManifest(t, ociRef, tmpDir)
+
+	// add the same manifest with a different tag; it shouldn't get overwritten
+	ociRef.image = "zomg"
+	putTestManifest(t, ociRef, tmpDir)
+
+	index, err := ociRef.getIndex()
+	assert.NoError(t, err)
+	assert.Equal(t, 3, len(index.Manifests), "Unexpected number of manifests")
```

Comment:
Absolutely non-blocking:
```suggestion
	assert.Len(t, index.Manifests, 3)
```

Reply:
Sure, I just copied this from somewhere else, so I fixed that location up too.****

Reply:
Thanks